archiver/main.go

```diff
@@ -21,148 +20,30 @@ import (
-	Name       string `json:"name"`
-	Groups     []struct {
-		GroupID int    `json:"group_id"`
-		Name    string `json:"name"`
-	} `json:"groups"`
-	PhishPronePercentage float64   `json:"phish_prone_percentage"`
-	StartedAt            time.Time `json:"started_at"`
-	Duration             int       `json:"duration"`
-	Categories           []struct {
-		CategoryID int    `json:"category_id"`
-		Name       string `json:"name"`
-	} `json:"categories"`
-	Template struct {
-		ID   int    `json:"id"`
-		Name string `json:"name"`
-	} `json:"template"`
-	LandingPage struct {
-		ID   int    `json:"id"`
-		Name string `json:"name"`
-	} `json:"landing-page"`
-	ScheduledCount        int `json:"scheduled_count"`
-	DeliveredCount        int `json:"delivered_count"`
-	OpenedCount           int `json:"opened_count"`
-	ClickedCount          int `json:"clicked_count"`
-	RepliedCount          int `json:"replied_count"`
-	AttachmentOpenCount   int `json:"attachment_open_count"`
-	MacroEnabledCount     int `json:"macro_enabled_count"`
-	DataEnteredCount      int `json:"data_entered_count"`
-	VulnerablePluginCount int `json:"vulnerable_plugin_count"`
-	ExploitedCount        int `json:"exploited_count"`
-	ReportedCount         int `json:"reported_count"`
-	BouncedCount          int `json:"bounced_count"`
-}
+const (
+	s3RecipientsFilenamePrefix = "knowbe4_recipients_"
+	phishingTestsFilename      = "campaigns/pst/knowbe4_security_tests.json"
+	campaignsFilename          = "campaigns/knowbe4_campaigns.json"
+)
 
-type KnowBe4FlatSecurityTest struct {
-	CampaignID            int       `json:"campaign_id"`
-	PstID                 int       `json:"pst_id"`
-	Status                string    `json:"status"`
-	Name                  string    `json:"name"`
-	Groups                string    `json:"all_groups"`
-	PhishPronePercentage  float64   `json:"phish_prone_percentage"`
-	StartedAt             time.Time `json:"started_at"`
-	Duration              int       `json:"duration"`
-	Categories            string    `json:"all_categories"`
-	TemplateID            int       `json:"template_id"`
-	TemplateName          string    `json:"template_name"`
-	LandingPageID         int       `json:"landing_page_id"`
-	LandingPageName       string    `json:"landing_page_name"`
-	ScheduledCount        int       `json:"scheduled_count"`
-	DeliveredCount        int       `json:"delivered_count"`
-	OpenedCount           int       `json:"opened_count"`
-	ClickedCount          int       `json:"clicked_count"`
-	RepliedCount          int       `json:"replied_count"`
-	AttachmentOpenCount   int       `json:"attachment_open_count"`
-	MacroEnabledCount     int       `json:"macro_enabled_count"`
-	DataEnteredCount      int       `json:"data_entered_count"`
-	VulnerablePluginCount int       `json:"vulnerable_plugin_count"`
-	ExploitedCount        int       `json:"exploited_count"`
-	ReportedCount         int       `json:"reported_count"`
-	BouncedCount          int       `json:"bounced_count"`
-}
+const (
```

Comment:
Any reason for multiple const blocks or just for clarity?

Reply:
Just for clarity. They are grouped logically.